decidim-conferences/app/cells/decidim/conferences/partner_cell.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Conferences
+    # This cell renders the partners card for an instance of a Partner
+    class PartnerCell < Decidim::ViewModel
+      def show
+        render
+      end
+
+      private
+
+      def partner
+        return block unless model.link.presence
```

Comment:
Maybe some of this view logic should go to the template, not inside the cell? It feels weird to me having all the markup here instead of the template.